contracts/NXMDSValue.sol

```diff
@@ -0,0 +1,39 @@
+/* Copyright (C) 2020 NexusMutual.io
+
+  This program is free software: you can redistribute it and/or modify
+    it under the terms of the GNU General Public License as published by
+    the Free Software Foundation, either version 3 of the License, or
+    (at your option) any later version.
+
+  This program is distributed in the hope that it will be useful,
+    but WITHOUT ANY WARRANTY; without even the implied warranty of
+    MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+    GNU General Public License for more details.
+
+  You should have received a copy of the GNU General Public License
+    along with this program.  If not, see http://www.gnu.org/licenses/ */
+
+pragma solidity 0.5.7;
+
+
+contract Aggregator {
+    function currentAnswer() public view returns (uint); 
+}
+
+
+contract NXMDSValue {
+
+    /// @dev Gets DAI feed address from chainlink feed and converts into desired format.
+    /// @return It return DAI Feed rate in 10^18 format.  
```

Comment:
// @dev Get ETH-USD feed from Chainlink and convert it to bytes32.
// @return Returns ETH-USD rate in wei.